packages/core/useTransition/index.ts

```diff
@@ -172,6 +179,13 @@ export function useTransition(
 
     resume()
     onStarted()
+  }
+
+  const timeout = useTimeoutFn(start, delay)
```

Comment:
Should we pass `immediate: false` here?

Reply:
Good catch! Because of the way I wrote my tests, I didn't notice this.

Resolved 345d8c1bd377b48a586be19e4819b66d1aa53856